vm/vm_test.go

```diff
@@ -43,9 +42,7 @@ func TestBlockCache(t *testing.T) {
 	controller := NewMockController(ctrl)
 	vm := VM{
 		snowCtx: &snow.Context{Log: logging.NoLog{}, Metrics: metrics.NewPrefixGatherer()},
-		config:  &config.Config{},
-
-		vmDB: memdb.New(),
+		vmDB:    memdb.New(),
```

Comment:
Can we populate this with a default config value instead of leaving it empty?

The previous code would have used the "defaults" since the `*config.Config` type would just always return the same values.